app/models/requests/patron.rb

```diff
@@ -94,7 +94,10 @@ def to_h
 
       def load_patron(user:)
         patron_hash = current_patron_hash(user.uid)
-        errors << "A problem occurred looking up your library account." if patron_hash.blank?
+        if patron_hash.instance_of?(TrueClass) || patron_hash.blank?
```

Comment:
For me, it would make more sense if we caught this error lower down in the stack trace, and make sure that this method doesn't get a patron_hash that's an instance of TrueClass.

Reply:
+1